src/snowflake/cli/_plugins/git/manager.py

```diff
@@ -27,16 +27,21 @@
 from snowflake.cli.api.identifiers import FQN
 from snowflake.connector.cursor import SnowflakeCursor
 
+OMIT_FIRST = slice(1, None)
```

Comment:
Can you add comment explaining what is the reasoning behind those slices?

Reply:
Reason was to replace magic numbers `[3:]`, `[1:]'.